README.md

```diff
@@ -1,56 +1,68 @@
 ![](https://github.com/OpenLiberty/open-liberty/blob/master/logos/logo_horizontal_light_navy.png)
-
 # Cloudant Sample
 This sample shows how to store data with Cloudant using CDI and MicroProfile Config, as well as data validation with Jakarta Bean Validation.
-
 ## Environment Set Up
 To run this sample, first [download](https://github.com/OpenLiberty/sample-cloudant/archive/main.zip) or clone this repo - to clone:
 ```
 git clone git@github.com:OpenLiberty/sample-cloudant.git
 ```
-
-### Setup Cloudant / CouchDb
-
-<!-- TODO explain setting up cloudant or couchdb -->
-
+### Setup CouchDb
+You will also need a CouchDB instance to use this sample. If you have Docker installed, you can use the following:
+```
+docker run -d --name liberty_cloudant -p 5984:5984 -e COUCHDB_USER=admin -e COUCHDB_PASSWORD=password couchdb:3.4.2
+```
 ## Running the Sample
 From inside the sample-cloudant directory, build and start the application in Open Liberty with the following command:
 ```
 ./mvnw liberty:dev
 ```
-
 Once the server has started, the application is available at http://localhost:9080
-
 ### Try it out
 Give the sample a try by registering a crew member. Enter a name (a String), an ID Number (an Integer), and select a Rank from the menu, then click 'Register Crew Member'.
-
 Two more boxes will appear, one with your crew members (which you can click to remove) and one showing how your data looks in Cloudant.
-
-### Stop Cloudant / CouchDb
-
-<!-- TODO instructions for stopping cloudant -->
-
+### Stop CouchDb
+```
+docker stop liberty_cloudant
+```
 ### How it works
+This application uses a CDI producer ([CloudantProducer.java](https://github.com/OpenLiberty/sample-cloudant/blob/main/src/main/java/io/openliberty/sample/cloudant/CloudantProducer.java)) to inject a CouchDatabase. It provides access to the database in a RESTful manner in [CrewService.java](https://github.com/OpenLiberty/sample-cloudant/blob/main/src/main/java/io/openliberty/sample/application/CrewService.java) using the `/db/crew` endpoint.
```

Comment:
```suggestion
This application uses a CDI producer ([CloudantProducer.java](https://github.com/OpenLiberty/sample-cloudant/blob/main/src/main/java/io/openliberty/sample/cloudant/CloudantProducer.java)) to inject a Cloudant client. It provides access to the database in a RESTful manner in [CrewService.java](https://github.com/OpenLiberty/sample-cloudant/blob/main/src/main/java/io/openliberty/sample/application/CrewService.java) using the `/db/crew` endpoint.
```

Slight difference from the mongo app, we can't inject a specific database access object, just the full Cloudant client